2018-03-28/azblob/zt_examples_test.go

```diff
@@ -1015,7 +1018,8 @@ func ExampleBlobUrl_Download() {
 
 	// Create a BlobURL object to a blob in the container (we assume the container & blob already exist).
 	u, _ := url.Parse(fmt.Sprintf("https://%s.blob.core.windows.net/mycontainer/BigBlob.bin", accountName))
-	blobURL := azblob.NewBlobURL(*u, azblob.NewPipeline(azblob.NewSharedKeyCredential(accountName, accountKey), azblob.PipelineOptions{}))
+	credential, _ := azblob.NewSharedKeyCredential(accountName, accountKey)
```

Comment:
given this is an example (and thus a recommended way of using this library) - shouldn't this be checking that the error object returned is nil?